src/core/config/clientConfig.js

```diff
@@ -0,0 +1,5 @@
+const clientConfig = {
```

Comment:
Just following up on the conversation we had,
I agree with you about adding a short comment explaining the purpose of the config here!